app/src/main/java/pl/ludek/smat/home/pool_office_client/data/apiservice/DataCallAdapter.kt

```diff
@@ -3,14 +3,14 @@ package pl.ludek.smat.home.pool_office_client.data.apiservice
 import retrofit2.*
 import java.lang.reflect.Type
 
-class DataCallAdapter <R : Any>(private val responseType: Type) :
+class DataCallAdapter<R : Any>(private val responseType: Type) :
```

Comment:
The main goal of introducing `BaseResponse` is to have a basic response class for the whole communication with the pool server. With such a class we get a simpler access to the data. In order to get the most from this the `R` should be a sub-type of `BaseResponse` like this `DataCallAdapter <R : BaseResponse>`